lib/Helper/Mail/Attachment.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace SendGrid\Helper\Mail;
+
+/**
+ * Class Attachment
+ * @package SendGrid\Helper
+ */
+class Attachment implements \JsonSerializable
+{
```

Comment:
If a user needs an `Attachment`, why can't we use the `constructor` to set the properties?, also, I don't see any type hint in the code, this feature has been able since `5.1` version and we have a base compatibility with `5.6`, and no doc blocks, if I want to contribute, I need to know what values I expect.

instead of `setContent($content` what about `setContent(Content $content)`, or better yet:

```php
public function __construct(Content $content, Type $type... etc)
```